sections/advanced/server-side-rendering.js

```diff
@@ -48,16 +48,18 @@ const ServerSideRendering = () => md`
 
   > \`sheet.getStyleTags()\` and \`sheet.getStyleElement()\` can only be called after your element is rendered. As a result, components from \`sheet.getStyleElement()\` cannot be combined with \`<YourApp />\` into a larger component.
 
+  You'll also need to customize the \`.babelrc\` and use \`babel-plugin-styled-components\`. Refer to the [tooling documentation](https://www.styled-components.com/docs/tooling#serverside-rendering) for more information.
```

Comment:
Let's change the wording a little, as this sounds like a requirement, which it isn't.

Since we're on this it'd be great to explain what the purpose of it is instead of instructing the user to use it. Things won't immediately break in its absence so it's important to understand why it might be necessary.

Maybe something along the lines of:

> You might need to install and use our babel plugin with its `ssr` option turned on.
> This prevents checksum mismatches by adding a deterministic ID to each styled component.

(There's no need to mention the details i.e. babelrc / plugin name since that's in the other section)

May I also ask you to please use a relative link and not a full URL, e.g. `/docs/tooling...`